shotover-proxy/src/transforms/mod.rs

```diff
@@ -81,7 +80,6 @@ pub enum Transforms {
     RedisTimestampTagger(RedisTimestampTagger),
     RedisSinkCluster(RedisSinkCluster),
     RedisClusterPortsRewrite(RedisClusterPortsRewrite),
-    #[cfg(test)]
     DebugReturner(DebugReturner),
```

Comment:
If we are adding this to our public API we will need to document it.

Reply:
Looking at the returner API:
```yaml
    - DebugReturner:
        response:
          Redis: "42"
```

The `response:` line is just noise.
Can we remove it?
I think we could do this by one of:
*  writing the type as `struct DebugReturner (Response)`
*  adding a serde attribute to merge with the parent.

Reply:
Raised #370